static/css/around-map.css

```diff
@@ -52,32 +52,33 @@ main {
 }
 
 .card h2 {
-  font-size: 2rem;
-}
-
-.card h3 {
-  font-size: 1.85rem;
-  font-weight: normal;
-  margin: 0 1.25rem 0 0;
+  font-size: 1.7rem;
+  margin: .75rem 0;
+  max-height: 5rem;
+  width: 27rem;
```

Comment:
Fixed width for any size? `100%` wouldn't do the job?
Flexbox doesn't offer anything?